src/vss.rs

```diff
@@ -0,0 +1,248 @@
+pub const VSS_SS_PRECOMMITTED: ::INT               = 0x05;
+pub const VSS_SS_PROCESSING_COMMIT: ::INT          = 0x06;
+pub const VSS_SS_COMMITTED: ::INT                  = 0x07;
+pub const VSS_SS_PROCESSING_POSTCOMMIT: ::INT      = 0x08;
+pub const VSS_SS_PROCESSING_PREFINALCOMMIT: ::INT  = 0x09;
+pub const VSS_SS_PREFINALCOMMITTED: ::INT          = 0x0a;
+pub const VSS_SS_PROCESSING_POSTFINALCOMMIT: ::INT = 0x0b;
+pub const VSS_SS_CREATED: ::INT                    = 0x0c;
+pub const VSS_SS_ABORTED: ::INT                    = 0x0d;
+pub const VSS_SS_DELETED: ::INT                    = 0x0e;
+pub const VSS_SS_POSTCOMMITTED: ::INT              = 0x0f;
+pub const VSS_SS_COUNT: ::INT                      = 0x10;
+
+pub type VSS_VOLUME_SNAPSHOT_ATTRIBUTES = ::DWORD;
+pub type PVSS_VOLUME_SNAPSHOT_ATTRIBUTES = *mut VSS_VOLUME_SNAPSHOT_ATTRIBUTES;
+pub const VSS_VOLSNAP_ATTR_PERSISTENT: ::DWORD           = 0x00000001;
+pub const VSS_VOLSNAP_ATTR_NO_AUTORECOVERY: ::DWORD      = 0x00000002;
+pub const VSS_VOLSNAP_ATTR_CLIENT_ACCESSIBLE: ::DWORD    = 0x00000004;
+pub const VSS_VOLSNAP_ATTR_NO_AUTO_RELEASE: ::DWORD      = 0x00000008;
+pub const VSS_VOLSNAP_ATTR_NO_WRITERS: ::DWORD           = 0x00000010;
+pub const VSS_VOLSNAP_ATTR_TRANSPORTABLE: ::DWORD        = 0x00000020;
+pub const VSS_VOLSNAP_ATTR_NOT_SURFACED: ::DWORD         = 0x00000040;
+pub const VSS_VOLSNAP_ATTR_NOT_TRANSACTED: ::DWORD       = 0x00000080;
+pub const VSS_VOLSNAP_ATTR_HARDWARE_ASSISTED: ::DWORD    = 0x00010000;
+pub const VSS_VOLSNAP_ATTR_DIFFERENTIAL: ::DWORD         = 0x00020000;
+pub const VSS_VOLSNAP_ATTR_PLEX: ::DWORD                 = 0x00040000;
+pub const VSS_VOLSNAP_ATTR_IMPORTED: ::DWORD             = 0x00080000;
+pub const VSS_VOLSNAP_ATTR_EXPOSED_LOCALLY: ::DWORD      = 0x00100000;
+pub const VSS_VOLSNAP_ATTR_EXPOSED_REMOTELY: ::DWORD     = 0x00200000;
+pub const VSS_VOLSNAP_ATTR_AUTORECOVER: ::DWORD          = 0x00400000;
+pub const VSS_VOLSNAP_ATTR_ROLLBACK_RECOVERY: ::DWORD    = 0x00800000;
+pub const VSS_VOLSNAP_ATTR_DELAYED_POSTSNAPSHOT: ::DWORD = 0x01000000;
+pub const VSS_VOLSNAP_ATTR_TXF_RECOVERY: ::DWORD         = 0x02000000;
+pub const VSS_VOLSNAP_ATTR_FILE_SHARE: ::DWORD           = 0x04000000;
+
+pub type VSS_SNAPSHOT_CONTEXT = ::DWORD;
+pub type PVSS_SNAPSHOT_CONTEXT = *mut VSS_SNAPSHOT_CONTEXT;
+pub const VSS_CTX_BACKUP: ::DWORD                    = 0;
+pub const VSS_CTX_FILE_SHARE_BACKUP: ::DWORD         = VSS_VOLSNAP_ATTR_NO_WRITERS;
+pub const VSS_CTX_NAS_ROLLBACK: ::DWORD              = ( ( VSS_VOLSNAP_ATTR_PERSISTENT | VSS_VOLSNAP_ATTR_NO_AUTO_RELEASE ) | VSS_VOLSNAP_ATTR_NO_WRITERS );
+pub const VSS_CTX_APP_ROLLBACK: ::DWORD              = ( VSS_VOLSNAP_ATTR_PERSISTENT | VSS_VOLSNAP_ATTR_NO_AUTO_RELEASE );
+pub const VSS_CTX_CLIENT_ACCESSIBLE: ::DWORD         = ( ( ( VSS_VOLSNAP_ATTR_PERSISTENT | VSS_VOLSNAP_ATTR_CLIENT_ACCESSIBLE ) | VSS_VOLSNAP_ATTR_NO_AUTO_RELEASE ) | VSS_VOLSNAP_ATTR_NO_WRITERS );
```

Comment:
No need for all these parentheses. Also use line breaks to fit within 99 columns.